From 824895f821f8cf50338e0fd72972d24d37952af2 Mon Sep 17 00:00:00 2001
From: Pawit Pornkitprasan <p.pawit@gmail.com>
Date: Mon, 11 Nov 2013 10:47:51 +0700
Subject: [PATCH 05/15] sepolicy: treat fuseblk as sdcard_external

Allow fuse-mounted NTFS/exFAT file systems to be written to

Change-Id: I1492914dd269a305e27aba58e61064d853adf2bc
---
 common/private/file.te        | 2 ++
 common/private/genfs_contexts | 1 +
 2 files changed, 3 insertions(+)
 create mode 100644 common/private/file.te
 create mode 100644 common/private/genfs_contexts

diff --git a/common/private/file.te b/common/private/file.te
new file mode 100644
index 0000000..ea6af04
--- /dev/null
+++ b/common/private/file.te
@@ -0,0 +1,2 @@
+# Filesystems
+type fuseblk, sdcard_type, fs_type, mlstrustedobject;
diff --git a/common/private/genfs_contexts b/common/private/genfs_contexts
new file mode 100644
index 0000000..6152690
--- /dev/null
+++ b/common/private/genfs_contexts
@@ -0,0 +1 @@
+genfscon fuseblk / u:object_r:fuseblk:s0
-- 
2.7.4

